tests/test_simplates.py

```diff
@@ -38,14 +38,12 @@ def test_cant_implicitly_override_state(harness):
 
 
 def test_can_explicitly_override_state(harness):
-    output = harness.simple("[---]\n"
-        "class Output: body=media_type=charset=code=299\n"
-        "state['output'] = Output()\n"
-        "[---]\n"
-        "bar"
```

Comment:
Does this mean it's no longer possible to influence output from inside a simplate?